### DISCUSSION TITLE
Adhole auf einer zweiten Fritz!Box - Wie konfigurieren?

### DISCUSSION THREAD

**Discussion by cbs38139** [Freetz-NG/freetz-ng#919] [Allgemein]
Hi! Ich möchte eine zweite Fritz!Box nutzen, um adhole darauf laufen zu lassen.

Ich habe bereits Freetz mit adhole und dnsmasq erfolgreich auf der zweite Fritz!Box installiert. Nennen wir "secondary". Ich habe adhole laufen lassen, um die Liste heruntergealden. dnsmasq läuft und wird automatisch gestartet,

Auf der Haupt-Fritz!Box (nennen wir sie "primary"), habe ich die IP-Adresse der secondary fest gesetzt und unter DNS-Server eingetragen.

Bisher alles gut und keine Fehler. Aber: Ich sehe immer noch Werbung. Am Beispiel spiegel.de: die Seite lädt normal, dann passiert lange nichts, und dann lädt die Werbung plötzlich.

Habe ich alles richtig konfiguriert?

![Screenshot 2024-01-15 at 19 51 15](https://github.com/Freetz-NG/freetz-ng/assets/67190489/3db2e0c1-d23a-4c05-b987-09a30448221a)

![Screenshot 2024-01-15 at 19 51 55](https://github.com/Freetz-NG/freetz-ng/assets/67190489/cb1f8f91-249b-4a15-97d6-adb5300a9bcf)

**Comment by fda77** [comment]
Da gibt es sehr viel Möglichkeiten. Macht das Programm mit dem du testest dns-over-tls oder dns-over https? Die Clients bekommen auch als dns-ip die 2. Box? AVM hat noch ein neues Feature, falls der DNS nicht erreichbar ist wird irgend ein anderer genutzt. Es könnten auch noch was mi DNS-Cache vom client sein usw
